apisix/schema_def.lua

```diff
@@ -465,6 +465,10 @@ _M.service = {
         name = {type = "string", maxLength = 50},
         desc = {type = "string", maxLength = 256},
         script = {type = "string", minLength = 10, maxLength = 102400},
+        enable_websocket = {
```

Comment:
we need to update the doc about `service`

Reply:
I have updated the admin-api.md, please check it.